packages/stark-build/config/webpack-partial.common.js

```diff
@@ -80,7 +89,9 @@ module.exports = metadata => {
 						{
```

Comment:
Can you remove the comment about `// this probably doesn't load the tslint configuration we think?`?

Reply:
I think this is included in the #1233 PR. But maybe the commit should be merged with this branch so it's 1 PR?

Reply:
Yeah, I would say we can have one single PR with all the changes